src/installer/BeatPackageCompiler/Properties/launchSettings.json

```diff
@@ -29,6 +24,11 @@
       "commandName": "Project",
       "commandLineArgs": "--package=elastic-agent-8.11.4-windows-x86_64 -v --keep-temp-files",
       "workingDirectory": "$(SolutionDir)"
+    },
+    "winlogbeat 8.12.1": {
```

Comment:
What's this and why does it only change winlogbeat?

Reply:
this is just an IDE configuration for local debug...